### PR TITLE
Barroux translations of the French Ordo and Commune

### PR DESCRIPTION
**Pull request by luc--**
This pull request replaces the translation of the french Ordo from the website introibo.fr with the translation of the missal of the abbey of Barroux.

Old texts are in *.old files.

---
A correction of <code>missa/Latin/Commune/C6a.txt</code> has also been made.

### CODE REVIEW

File: web/www/missa/Latin/Commune/C6a.txt [hunk 1]
```diff
@@ -40,11 +40,12 @@ Altíssimus.
 Veni, Sponsa Christi, áccipe corónam, quam tibi Dóminus præparávit in ætérnum: pro cujus amóre sánguinem tuum fudísti
 !Ps 44:8; 44:5
 Dilexísti justítiam, et odísti iniquitátem proptérea unxit te Deus, Deus tuus, óleo lætítiæ præ consórtibus tuis.
-V. Spécie tua et pulchritúdine tua inténde, próspere procéde et regna. Tempore autem Paschali omittitur
+V. Spécie tua et pulchritúdine tua inténde, próspere procéde et regna.
+! Tempore paschali omittitur graduale.
```
Comment (gregordick) [inline web/www/missa/Latin/Commune/C6a.txt]: This line can be omitted altogether, as "GradualeP" below will be used automatically during Paschaltide.

**Comment by gregordick** [comment]
Thanks.  Could you please remove the `*.old` files?  We can recover the old texts from source control history if we ever need them.

**Comment by luc--** [comment]
Ok this is (finally) done with 313e9c1.

**Comment by gregordick** [comment]
Thanks -- merged.  I notice that 313e9c1 also makes a few other changes than removing the `*.old` files.  In general it would be better to have distinct changes such as those in separate commits.

**Comment by luc--** [comment]
Sorry, I thought I dit just a git rm on all the old files.

**Comment by gregordick** [comment]
No problem.  Do the changes to the other files need reverted, or can they stay?

**Comment by luc--** [comment]
No I've just verified, all is correct (changes on other files were in fact still consequences of the suppression of the old translation). Thanks !

**Pull request merged**